Makefile

```diff
@@ -2904,6 +2931,46 @@ distclean: clean toolsclean
 dist:
 	git archive --format zip --output $(CLIENTBIN)-$(VERSION).zip HEAD
 
+#############################################################################
+# INSTALL (only for Linux platforms)
+#############################################################################
+
+# Shell scripts for running binaries
+
+$(B)/$(CLIENTBINSH): 
+	@echo '#!/usr/bin/env sh' > $@
+	@echo 'cd $(COPYBINDIR)' >> $@
+	@echo './$(CLIENTBIN) "$$@"' >> $@
+
+$(B)/$(SERVERBINSH): 
+	@echo '#!/usr/bin/env sh' > $@
+	@echo 'cd $(COPYBINDIR)' >> $@
+	@echo './$(SERVERBIN) "$$@"' >> $@
+
+install: release
+ifneq (,$(findstring "$(PLATFORM)", "linux" "gnu_kfreebsd" "kfreebsd-gnu" "gnu"))
+	$(echo_cmd) "Installing for Linux platform in $(COPYBINDIR) and $(PREFIX)"
+	@$(MKDIR) -p $(COPYBINDIR)
+	@cd $(BR) && for file in $(NAKED_TARGETS); do \
+		$(INSTALL) -D $$file $(COPYBINDIR)/$$file; \
+	done
+#	@(cd $(BR) && $(INSTALL) $() $(COPYBINDIR) )
+# TODO: create these .sh files:
+	@$(INSTALL) -d $(PREFIX)/bin
+	@$(INSTALL) -D -m755 $(BR)/$(CLIENTBINSH) $(BR)/$(SERVERBINSH) $(PREFIX)/bin
+# Install the .desktop, icon files, license, etc.
+	@$(INSTALL) -D -m644 "misc/io.github.grangerhub.Tremulous.png" "$(PREFIX)/share/pixmaps/tremulous.png"
```

Comment:
```suggestion
	@$(INSTALL) -D -m644 "misc/io.github.grangerhub.Tremulous.png" "$(PREFIX)/share/icons/hicolor/128x128/apps/io.github.grangerhub.Tremulous.png"
```

Reply:
I think I already fixed these. Didn't know you could make suggestions here!